src/js/components/homepage/LinksSectionBox.jsx

```diff
@@ -41,16 +41,32 @@ export default class LinksSectionBox extends React.Component {
     }
 
     render() {
+        const arrow = (<Icons.AngleRight />);
```

Comment:
No real need for setting this as a variable - we can move `<Icons.AngleRight />` into the <a> below